Dockerfile-fpm

```diff
@@ -23,9 +23,10 @@ RUN set -x \
 COPY src/php/utils/docker/ /usr/local/bin/
 COPY src/php/utils/install-* /usr/local/bin/
 
-# Install PHP extensions
+# Install PHP extensions, install curl-dev in order to address the curl binary issue in some Alpine versions
 # hadolint ignore=DL4006
 RUN set -x \
+    && apk add --no-cache curl-dev \
```

Comment:
Would be good to keep the comment as close to the code as possible in my opinion for clarity - same for the other Dockerfile.
```suggestion
# Install PHP extensions
# hadolint ignore=DL4006
RUN set -x \
    # install curl-dev in order to address the curl binary issue in some Alpine versions
    && apk add --no-cache curl-dev \
```

Reply:
:+1: